Assets/Scripts/2D/GuiManagerScript.cs

```diff
@@ -1304,33 +1312,33 @@ public void RegenerateWorldRiverLevelOffsetChange(float value)
     public void RegenerateWorldTemperatureOffsetChange(float value)
     {
         Manager.TemperatureOffset = value;
-
-        RegenerateWorld(GenerationType.TemperatureRegeneration);
```

Comment:
Everything appropriately changed to TerrainRegeneration since whole world has to be regenerated.